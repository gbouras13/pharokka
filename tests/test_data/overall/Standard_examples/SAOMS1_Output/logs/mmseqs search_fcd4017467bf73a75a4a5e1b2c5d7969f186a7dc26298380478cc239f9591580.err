Command line: mmseqs search --min-seq-id 0.8 -c 0.4 ../../databases/CARD SAOMS1_Output/CARD_target_dir/target_seqs SAOMS1_Output/CARD/results_mmseqs SAOMS1_Output/VFDB_dir/ -s 8.5 --threads 8
