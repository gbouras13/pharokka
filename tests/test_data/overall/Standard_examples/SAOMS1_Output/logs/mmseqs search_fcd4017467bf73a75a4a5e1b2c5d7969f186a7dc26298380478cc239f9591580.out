Create directory SAOMS1_Output/VFDB_dir/
search --min-seq-id 0.8 -c 0.4 ../../databases/CARD SAOMS1_Output/CARD_target_dir/target_seqs SAOMS1_Output/CARD/results_mmseqs SAOMS1_Output/VFDB_dir/ -s 8.5 --threads 8 

MMseqs Version:                        	13.45111
Substitution matrix                    	nucl:nucleotide.out,aa:blosum62.out
Add backtrace                          	false
Alignment mode                         	2
Alignment mode                         	0
Allow wrapped scoring                  	false
E-value threshold                      	0.001
Seq. id. threshold                     	0.8
Min alignment length                   	0
Seq. id. mode                          	0
Alternative alignments                 	0
Coverage threshold                     	0.4
Coverage mode                          	0
Max sequence length                    	65535
Compositional bias                     	1
Max reject                             	2147483647
Max accept                             	2147483647
Include identical seq. id.             	false
Preload mode                           	0
Pseudo count a                         	1
Pseudo count b                         	1.5
Score bias                             	0
Realign hits                           	false
Realign score bias                     	-0.2
Realign max seqs                       	2147483647
Gap open cost                          	nucl:5,aa:11
Gap extension cost                     	nucl:2,aa:1
Zdrop                                  	40
Threads                                	8
Compressed                             	0
Verbosity                              	3
Seed substitution matrix               	nucl:nucleotide.out,aa:VTML80.out
Sensitivity                            	8.5
k-mer length                           	0
k-score                                	2147483647
Alphabet size                          	nucl:5,aa:21
Max results per query                  	300
Split database                         	0
Split mode                             	2
Split memory limit                     	0
Diagonal scoring                       	true
Exact k-mer matching                   	0
Mask residues                          	1
Mask lower case residues               	0
Minimum diagonal score                 	15
Spaced k-mers                          	1
Spaced k-mer pattern                   	
Local temporary path                   	
Rescore mode                           	0
Remove hits by seq. id. and coverage   	false
Sort results                           	0
Mask profile                           	1
Profile E-value threshold              	0.1
Global sequence weighting              	false
Allow deletions                        	false
Filter MSA                             	1
Maximum seq. id. threshold             	0.9
Minimum seq. id.                       	0
Minimum score per column               	-20
Minimum coverage                       	0
Select N most diverse seqs             	1000
Min codons in orf                      	30
Max codons in length                   	32734
Max orf gaps                           	2147483647
Contig start mode                      	2
Contig end mode                        	2
Orf start mode                         	1
Forward frames                         	1,2,3
Reverse frames                         	1,2,3
Translation table                      	1
Translate orf                          	0
Use all table starts                   	false
Offset of numeric ids                  	0
Create lookup                          	0
Add orf stop                           	false
Overlap between sequences              	0
Sequence split mode                    	1
Header split mode                      	0
Chain overlapping alignments           	0
Merge query                            	1
Search type                            	0
Search iterations                      	1
Start sensitivity                      	4
Search steps                           	1
Exhaustive search mode                 	false
Filter results during exhaustive search	0
Strand selection                       	1
LCA search mode                        	false
Disk space limit                       	0
MPI runner                             	
Force restart with latest tmp          	false
Remove temporary files                 	false

prefilter ../../databases/CARD SAOMS1_Output/CARD_target_dir/target_seqs SAOMS1_Output/VFDB_dir//6485957439078731740/pref_0 --sub-mat nucl:nucleotide.out,aa:blosum62.out --seed-sub-mat nucl:nucleotide.out,aa:VTML80.out -k 0 --k-score 2147483647 --alph-size nucl:5,aa:21 --max-seq-len 65535 --max-seqs 300 --split 0 --split-mode 2 --split-memory-limit 0 -c 0.4 --cov-mode 0 --comp-bias-corr 1 --diag-score 1 --exact-kmer-matching 0 --mask 1 --mask-lower-case 0 --min-ungapped-score 15 --add-self-matches 0 --spaced-kmer-mode 1 --db-load-mode 0 --pca 1 --pcb 1.5 --threads 8 --compressed 0 -v 3 -s 8.5 

Query database size: 4641 type: Aminoacid
Estimated memory consumption: 978M
Target database size: 246 type: Aminoacid
Index table k-mer threshold: 87 at k-mer size 6 
Index table: counting k-mers
[=================================================================] 246 0s 4ms
Index table: Masked residues: 369
Index table: fill
[=================================================================] 246 0s 3ms
Index statistics
Entries:          40114
DB size:          488 MB
Avg k-mer size:   0.000627
Top 10 k-mers
    LSPQEA	2
    GSSSEA	2
    SKSPKA	2
    SRGKAD	2
    VYKDDD	2
    SDMGDD	2
    LSYVED	2
    DDILID	2
    KRTNID	2
    YKYKLD	2
Time for index table init: 0h 0m 0s 332ms
Process prefiltering step 1 of 1

k-mer similarity threshold: 87
Starting prefiltering scores calculation (step 1 of 1)
Query db start 1 to 4641
Target db start 1 to 246
[=================================================================] 4.64K 16s 914ms

3883.681631 k-mers per position
819 DB matches per sequence
0 overflows
0 queries produce too many hits (truncated result)
16 sequences passed prefiltering per query sequence
14 median result list length
1 sequences with 0 size result lists
Time for merging to pref_0: 0h 0m 0s 6ms
Time for processing: 0h 0m 17s 656ms
align ../../databases/CARD SAOMS1_Output/CARD_target_dir/target_seqs SAOMS1_Output/VFDB_dir//6485957439078731740/pref_0 SAOMS1_Output/CARD/results_mmseqs --sub-mat nucl:nucleotide.out,aa:blosum62.out -a 0 --alignment-mode 2 --alignment-output-mode 0 --wrapped-scoring 0 -e 0.001 --min-seq-id 0.8 --min-aln-len 0 --seq-id-mode 0 --alt-ali 0 -c 0.4 --cov-mode 0 --max-seq-len 65535 --comp-bias-corr 1 --max-rejected 2147483647 --max-accept 2147483647 --add-self-matches 0 --db-load-mode 0 --pca 1 --pcb 1.5 --score-bias 0 --realign 0 --realign-score-bias -0.2 --realign-max-seqs 2147483647 --gap-open nucl:5,aa:11 --gap-extend nucl:2,aa:1 --zdrop 40 --threads 8 --compressed 0 -v 3 

Compute score and coverage
Query database size: 4641 type: Aminoacid
Target database size: 246 type: Aminoacid
Calculation of alignments
[=================================================================] 4.64K 0s 268ms
Time for merging to results_mmseqs: 0h 0m 0s 5ms
52224 alignments calculated
0 sequence pairs passed the thresholds (0.000000 of overall calculated)
0.000000 hits per query sequence
Time for processing: 0h 0m 0s 288ms
